examples/tube_fluent3d_abaqus2d/parameters.json

```diff
@@ -139,8 +139,9 @@
                     "type": "mappers.axisymmetric_3d_to_2d",
                     "settings": {
                       "direction_axial": "y",
-                      "direction_radial": "x",
-                      "n_tangential": 8
+                      "direction_radial":"x",
+	              "angle":360,
+                      "n_tangential": 10
```

Comment:
- _`examples/tube_fluent3d_abaqus2d/parameters.json`_ should not be changed (360 is default)